src/components/PreviewPane/PreviewPane.js

```diff
@@ -15,7 +15,8 @@ export default class PreviewPane extends React.Component {
   getWidget = (field, value, props) => {
     const { fieldsMetaData, getAsset } = props;
     const widget = resolveWidget(field.get('widget'));
-    return React.createElement(widget.preview, {
+
+    return !widget.preview ? null : React.createElement(widget.preview, {
```

Comment:
This should hopefully fix the boolean issue @verythorough was running into, thanks!